apps/web/src/api/stripe.ts

```diff
@@ -164,6 +164,40 @@ const findSessionPlan = async (initialSession: Stripe.Checkout.Session) => {
   return findPlanOrThrow(stripeProductId);
 };
 
+export const updateStripeUsage = async (account: Account) => {
+  try {
+    const stripeCustomerId = account.stripeCustomerId;
+    if (!stripeCustomerId) {
+      return 0;
+    }
+
+    const subscriptions = await stripe.subscriptions.list({
+      customer: stripeCustomerId,
+      expand: ["items"],
+      limit: 1,
+    });
+    const subscription = subscriptions.data[0];
+    const meteredPricing =
+      subscription?.items.data[0]?.price.recurring?.usage_type === "metered";
```

Comment:
If the user has several subscriptions?

Reply:
We don't provide multiple active subscription

Reply:
If prefer to request only `active` without `limit` if we found `> 1` then we throw.